ansible/roles/kamailio/templates/sipmediagw.cfg.j2

```diff
@@ -0,0 +1,10 @@
+[sip]
+  sipSrv="{{ ansible_host }}"
+  sipSecret="1234"
+  turnConfig="turn:{{ coturn_host }}:3478;stunuser=dmo;stunpass=1234"
+
+[mediagw]
+  sipUaNamePart="mediagw"
+  cpuCorePerGw=2.5
+  launcherAPIPath="http://{{ gwlauncher_host }}:8080/sipmediagw"
```

Comment:
This should be set in (default) variables: sipSecret, coturn port, stun user, stun password, whole launcherAPIPath URL